flambe/runnable/runnable.py

```diff
@@ -24,11 +25,13 @@ class Runnable(MappedRegistrable):
         'config["AWS"]["ACCESS_KEY"]'
 
     """
-    def __init__(self, **kwargs) -> None:
+    def __init__(self, user_provider: Callable[[], str] = None, **kwargs) -> None:
```

Comment:
Why None? can't the default be DEFAULT_USER_PROVIDER?

Reply:
Fixed.